src/style.css

```diff
@@ -1 +1,60 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+* {
```

Comment:
Using the universal selector `*` can impact performance negatively, especially on large documents. Consider using more specific selectors to improve performance.